build/bootstrap/bootstrap-debian.sh

```diff
@@ -70,3 +71,14 @@ sudo mv /tmp/bazelisk /usr/bin/bazel
 
 # Install the Unison file-syncer.
 . bootstrap/bootstrap-unison.sh
+
+# Configure environment variables for CockroachDB and Managed Service.
+echo 'export PATH="${PATH}:$HOME/go/src/github.com/cockroachdb/cockroach/bin:/usr/local/go/bin"' >> ~/.bashrc_bootstrap
+echo 'export PATH="${PATH}:$HOME/go/src/github.com/cockroachlabs/managed-service/bin"' >> ~/.bashrc_bootstrap
+echo '. ~/.bashrc_bootstrap' >> ~/.bashrc
+. ~/.bashrc_bootstrap
+
+git clone https://github.com/cockroachdb/cockroach "$(go env GOPATH)/src/github.com/cockroachdb/cockroach"
+git -C "$(go env GOPATH)/src/github.com/cockroachdb/cockroach" submodule update --init
+
+git clone git@github.com:cockroachlabs/managed-service.git "$(go env GOPATH)/src/github.com/cockroachlabs/managed-service"
```

Comment:
If the name of the file is _not_ `id_rsa` for the private key (or another standard key name), then cloning the managed-service repo will fail as written. Perhaps make a link to the file using the `id_rsa` name if the chosen filename is different?

Reply:
I am now creating a symlink of copied key to `~/.ssh/id_rsa`